src/components/dashboard/FaceRecognition/FaceRecognitionAPI.js

```diff
@@ -15,45 +14,29 @@ type FaceRecognitionAPIResponse = {
   ok: boolean,
   error: string,
 }
+type CaptureResult = { sessionId: String, images: Array<any> }
 
 const log = logger.child({ from: 'FaceRecognitionAPI' })
 
 /**
  * Responsible to communicate with GoodServer and UserStorage on FaceRecognition related actions, and handle sucess / failure
  * * onFaceRecognitionFailure: Analyze the failure reason and returns a proper error message
- * * createFaceRecognitionReq - prepares the FR request for the server
  * * performFaceRecognition - calls the server API to perform FR process
  * * onFaceRecognitionResponse - Analyze the server result and call failure / success handler accordingly
  * * onFaceRecognitionSuccess - sets the enrollmentIdentifier (recivied from a successful FR process) on the user private storage
  */
 export const FaceRecognitionAPI = {
```

Comment:
1. Why not a class like lib/API/api ? If it will be a class, you could define onFaceRecognitionResponse and other architecturally non-public methods as `private`
2. But generally you have just one function with simple logic. You don't need objects/classes with separate onXXXResponse/Success/Failure methods. Everything can be declared in a single simple function which can be exported from the module

Reply:
done, export only **performFaceRecognition**